src/components/side_panel/chart/chart_with_axis/design_panel.xml

```diff
@@ -9,7 +9,7 @@
           <t t-set-slot="title">Legend position</t>
           <select
             t-att-value="props.definition.legendPosition ?? 'top'"
-            class="o-input"
```

Comment:
The class `legend-position-selector` doesn't seems to be used anywhere I think ?